ReactWindows/ReactNative.Tests/project.json

```diff
@@ -1,7 +1,8 @@
 ﻿{
   "dependencies": {
     "Facebook.CSSLayout": "2.0.1-pre",
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2"
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
+    "NUnit": "3.4.1"
   },
```

Comment:
Is this dependency being used yet?  I don't see any dependencies on ReactNative.Shared.Tests in ReactNative.Tests.csproj